### PR TITLE
Added our meetup at MCH 2022

### PR DESCRIPTION
**Pull request by poef**

### CODE REVIEW

**Comment by Vinnl** [comment]
Thanks @poef. Is there a page on the camp website about the meetup specifically? Would be nice if you could link that instead, as I can't find it via the homepage...

**Comment by poef** [comment]
I've just added it, apparantly I'm the first to add a self organized session, jay!

https://wiki.mch2022.org/Session:Solid,_Linked_Data_and_Decentralized_Web_Meetup

File: pages/solid-events.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |------|-------|-----------|-----------------------------|
+|2022-07-22| [MCH 2022](https://mch2022.org/) | [Muze](https://muze.nl) | Muze hosts a meetup about Solid, Linked Data and Decentralized Web at MCH 2022, in Zeewolde, the Netherlands. There will be talks and workshops on three afternoons, starting at July 23th. If you've got something to show or tell, or just tell us that you're coming, please mail to [mch@muze.nl](mailto:mch@muze.nl). |
```
Comment (Vinnl) [inline pages/solid-events.md:19]: Thanks! How about this:

```suggestion
|2022-07-22| [Solid, Linked Data and Decentralized Web Meetup](https://wiki.mch2022.org/Session:Solid,_Linked_Data_and_Decentralized_Web_Meetup) | [MCH 2022](https://mch2022.org/) / [Muze](https://muze.nl) | Muze hosts a meetup about Solid, Linked Data and Decentralized Web at MCH 2022, in Zeewolde, the Netherlands. There will be talks and workshops on three afternoons, starting at July 23th. If you've got something to show or tell, or just tell us that you're coming, please mail to [mch@muze.nl](mailto:mch@muze.nl). |
```

**Comment by Vinnl** [comment]
Thank you!

**Pull request merged**